### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by MatthewRiley05** [Floorp-Projects/Floorp#1080]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [X] I have tried clearing the browser cache and cookies.
- [X] I have tried to create a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [X] I have included relevant screenshots or console outputs.
- [X] I have checked this problem is not ocurring in the latest version of Firefox ESR.
- [X] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

Webpages and user interface are laggy and choppy when windowed fullscreen but entirely smooth when fullscreen or smaller windowed.

### Reproduction Steps

1. Open browser
2. click the upper right icon to windowed fullscreen

### Expected Behavior

Supposedly windowed, windowed fullscreen, and fullscreen are supposed to be fully smooth without lag and latency.,

### Actual Behavior

The windowed fullscreen setting adds some sort of lagginess to the software and there is also a feeling of delayed mouse or keyboard actions.

### Screenshots

https://github.com/Floorp-Projects/Floorp/assets/52445584/9b40ab26-7cd6-46d3-8673-e7b67c4fab9c

### Browser Version

Version 11.11.2 (64-bit) | Firefox: 115.10.0

### Operating System

Windows 11

### Additional Context

One crucial detail to note is hardware acceleration is turned off because it makes YouTube videos laggy, and the frame stays still for moments at various portions of the video. The YouTube website also sometimes becomes unresponsive, not just the video. Turning off hardware acceleration completely solves this issue.

**Comment by TheChilledBuffalo** [comment]
Can't reproduce this 🤔

I've tried it with Hardware Acceleration on and off. Nothing seems off on my end... Is this reproducible on a fresh new profile for you?

**Comment by TheChilledBuffalo** [comment]
And also try disabling background tasks that might be using high CPU or GPU... Because this issue seems to be specific to your environment and I don't thing anything can be done from Floorp's side...

**Comment by MatthewRiley05** [comment]
No, I'm using a pretty high-specced laptop running an AMD Ryzen 9 7940HS and an RTX 4070 with 48GB of RAM. But yes, looking more into this issue made me realize that it is probably more of a local issue than a Floorp issue; I will be closing the issue. Thank you for your time.
